src/Uno.Toolkit.UI/Behaviors/StatusBar.cs

```diff
@@ -222,15 +222,17 @@ private static void OnUISettingsColorValuesChanged(UISettings sender, object arg
 		{
 			if (_lastActivePage is { } page)
 			{
-				if (GetSubscription(page) != null &&
+				page.GetDispatcherCompat().Schedule(() => {
```

Comment:
we can probably do without scheduling on non-windows targets:
```suggestion
#if !HAS_UNO
				page.GetDispatcherCompat().Schedule(() => {
#else
				{
#endif
```

Reply:
I added Windows-specific conditionals